ledger/sandbox/BUILD.bazel

```diff
@@ -32,6 +32,7 @@ compile_deps = [
     "//ledger/ledger-api-domain",
     "//ledger/ledger-api-health",
     "//ledger/ledger-on-sql",
+    "//ledger/metrics",
```

Comment:
Would it make sense to put it in `//libs-scala/` instead?

Reply:
Right now, yes, but I am expecting it to get more niche. I want to add https://github.com/vgangan/daml/pull/1 to this directory too. That PR contains a lot of Grafana configuration that's specific to ledgers.